README.md

```diff
@@ -38,6 +42,56 @@ make cloudbuild
 
 The Cloud Build steps are defined in [`cloudbuild.yaml`](cloudbuild.yaml).
 
+## Running on GKE
+
+### Prerequisites
+
+* Running GKE cluster with [Workload Identity](https://cloud.google.com/kubernetes-engine/docs/how-to/workload-identity) enabled
```

Comment:
Every time I've tried running the collector from scratch in GKE I have needed to set up workload identity, so I think it's required

Reply:
I don't believe it is required... It is enabled by default in autopilot clusters, though.  Are you using one of those?

The reference for --workload-pool suggests it is disabled by default: https://cloud.google.com/sdk/gcloud/reference/container/clusters/create#--workload-pool

Reply:
🤷 enabling workload identity is the only way I have been able to get collectors in GKE to send telemetry to GCP, otherwise I get permission errors on export

Reply:
I created a new GKE cluster, followed all steps except workload-identity-related steps, and it worked fine.  We can sync up on this when I'm back.

Reply:
I'm not sure then. Moved these steps to their own "troubleshooting" section for now, but I think we should re-try this with a fresh GCP project.